README.md

```diff
@@ -782,10 +782,10 @@ end
 ```ruby
 class User
   def custom_columns
-    self.dynamic_fields #which is an array of some columns
+    # which is an array of some columns
   end
 
-  def custom_fields
+  def dynamic_fields
```

Comment:
aligned to `hash.merge!(object.dynamic_fields)` in [here](https://github.com/procore-oss/blueprinter/pull/415/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R777)